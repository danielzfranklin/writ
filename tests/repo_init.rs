mod support;
use support::assert_eq;
use support::*;

#[test]
fn can_init() -> Result {
    init();
    let (dir, _repo) = repo_fixture()?;
    assert_debug_snapshot!(all_entries(dir.path().join(".git"))?);

    Ok(())
}
