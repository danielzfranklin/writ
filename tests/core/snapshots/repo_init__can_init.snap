---
source: tests/repo_init.rs
expression: "all_entries(dir.path().join(\".git\"))?"

---
[
    "objects",
    "refs",
]
